code/modules/mob/living/carbon/xenomorph/abilities/xeno_action.dm

```diff
@@ -144,7 +146,7 @@
 /datum/action/xeno_action/activable
 
 /datum/action/xeno_action/activable/can_use_action()
-	return TRUE
+	. = ..()
```

Comment:
If this was code we wanted, you'd just omit it because it effectively does nothing but just create unneeded proc overhead.

---

This is not an improvement:
New: 
![image](https://github.com/user-attachments/assets/48d5deab-cf73-42f2-8008-12645d602018)
Old:
![image](https://github.com/user-attachments/assets/9937c28e-9982-4dc9-bab1-e48367345176)
They should still be selectable because no action is performed when selecting them. It just selects them.

---
This also makes non t1s disable all their abilities other than tail stab when stalking.

---

Consider instead merely checking the stalk state & tier here rather than parent implementation.

Reply:
If you want to see the blame for this particular line, it is very much intended: https://github.com/cmss13-devs/cmss13/commit/f694746fb70936ebc6be69a21e23a96e96bc2379

Reply:
will no longer call parent . it will now only check for stalking mode and if the owner is alive